react-form-controls/my-app/src/RegistrationFormUncontrolled.js

```diff
@@ -0,0 +1,23 @@
+export default function RegistrationFormUncontrolled() {
+  function handleSumbit(event) {
+    event.preventDefault();
+    const formData = new FormData(event.target);
+
+    const state = Object.fromEntries(formData.entries());
```

Comment:
Looks like you will need to look at step 5 again. Instead of a variable `state` you will destructure `const {username, password} ` and then log those values.

**When the form is submitted, log the form values to the console, as { username, password }. To do this, you will need to use [FormData](https://developer.mozilla.org/en-US/docs/Web/API/FormData) to get the form content, and Object.fromEntries(formData.entries()) to get the individual field values.**